client/templates/initiatives/mine/mine.html

```diff
@@ -14,7 +14,7 @@
       {{/each}}
     </div>
     <div class="initiative-mine row center-align">
-        <a class="button btn" href="{{pathFor 'initiatives.create'}}">New Initiative</a>
+        <a class="button waves-effect waves-teal btn-flat" href="{{pathFor 'initiatives.create'}}">New Initiative</a>
```

Comment:
Sure thing, i think the buttons are hideous lol eventually i'll do a complete design... but for now its ok